multimodal/tests/unittests/test_object_detection.py

```diff
@@ -35,7 +35,7 @@ def test_mmdet_object_detection_inference(checkpoint_name):
         pipeline="object_detection",
     )
 
-    pred = predictor.predict({"image": [mmdet_image_name] * 10}, realtime=False)  # test batch inference
+    pred = predictor.predict({"image": [mmdet_image_name] * 10})  # test batch inference
```

Comment:
Is the default for realtime still False or True?

Reply:
The default is None. We infer whether True or Flase.